tools/mac_setup.sh

```diff
@@ -15,12 +21,31 @@ brew install capnp \
              pyenv \
              zeromq
 
-# install python
+# Detect shell and pick correct RC file.
+if [[ $SHELL == "/bin/zsh" ]]; then
+  RC_FILE="$HOME/.zshrc"
+elif [[ $SHELL == "/bin/bash" ]]; then
+  RC_FILE="$HOME/.bash_profile"
+else
+  echo "-------------------------------------------------------------"
+  echo "Unsupported shell: \"$SHELL\", cannot install to RC file."
+  echo "Please run: echo \"source $OP_DIR/tools/openpilot_env.sh\" >> %YOUR SHELL's RC file%"
+  echo "-------------------------------------------------------------"
+fi
+
+# Install to RC file.
+source ~/.zshrc
```

Comment:
this is unnecessary

Reply:
This was present on the ubuntu_setup.sh, should it also be removed there?